docs/recipes/Bundling-Modules.md

```diff
@@ -0,0 +1,186 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Bundling
+
+Within `one-app` you can bundle your modules once you've made changes to them. The bundled files are then loaded on the client and server.
+
+## Bundling Modules
+
+Generate your module using [generator-one-app-module](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module).
+
+The module will have the following folder structure. Ensure it follows the pattern below for the build files to be generated correctly:
+
+
+Module:
+```
+
+module
+├── README.md
+├── package.json
+├── src
+|   ├── components
+|   |    └── <NameOfComponent.jsx>
+|   |
+|   |
+|   └── index.js
+```
+
+> If doing local development ensure that you've set the `process.env.NODE_ENV` to development using the below script by default webpack set its to production automatically.Please refer to [webpack mode](https://webpack.js.org/?configuration/mode/) for more information.
+
+ ``` sh
+
+ export NODE_ENV=development
+
+ ```
+
+When the [build commands `npm run build`](#build-commands) is executed on the above root folder the following files are generated within the module. The root folder of the build would be the version number provided within the package.json.
+
+Module:
+```
+
+module
+├── README.md
+├── package.json
+├── src
+|   ├── components
+|   |    └── <NameOfComponent.jsx>
+|   |
+|   └── csp.js
```

Comment:
`csp.js` is missing on the previous structure before line 25, which might imply that the `npm run build` command generates that file. We should remove it if considered irrelevant for this example or keep it as it is the default file structure for a freshly created module using generator...

Reply:
thank you for catching this, i've updated this 6dd6088